src/test/app/forms/models/summary.ts

```diff
@@ -0,0 +1,85 @@
+/* Allow chai assertions which don't end in a function call, e.g. expect(thing).to.be.undefined */
+/* tslint:disable:no-unused-expression */
+
+import { expect } from 'chai'
+import { Validator } from 'class-validator'
+import { expectValidationError } from './validationUtils'
+import Summary, { ValidationErrors } from 'forms/models/summary'
+import * as randomstring from 'randomstring'
+
+describe('Reason', () => {
+  describe('constructor', () => {
+    it('should set the primitive fields to undefined', () => {
+      const reason = new Summary()
+      expect(reason.summary).to.be.undefined
```

Comment:
Will recommend to rename `Summary.summary` field to `Summary.text` or `Summary.reason` something similar.